### DISCUSSION TITLE
Optimization Model Functionality

### DISCUSSION THREAD

**Discussion by blunte3** [project-pareto/project-pareto#243] [Q&A]
I've been working on a private copy of the repository and testing my own excel data. I've been messing around with toy_case_study and it has been working well but in the overview it says it can output solutions that are more specific than just drive times and drive distances. I've been looking and I can't really find this for the optimization model, is this something I must add or is there already pre-existing code that works with the data I have to run to get specific outputs to match together? I'm also wondering what other possibilities this tool can create. Thank you!

**Comment by tarnold17** [comment]
@blunte3 - can you clarify what you mean by "in the overview it says it can output solutions that are more specific than just drive times and drive distances"? Where are you seeing that?

**Reply by blunte3** [reply]
The overview in the case study .xlsx files on the first sheet, it shows the pathflow of the optimization model as well as an example output that is a possibility and I was having a hard time finding where I could generate this output or if it existed. No issue now though I've been figuring it out as slowly as I mess around with the code I cloned. Thank you!

**Reply by tarnold17** [reply]
@blunte3 I am glad to hear that! My apologies that it took two weeks to respond to this inquiry. Please don't hesitate to reach out with any further questions!
